src/js/rx/reducers/prescriptions.js

```diff
@@ -91,7 +94,7 @@ export default function prescriptions(state = initialState, action) {
       if (action.active) {
         newState.active = {
           loading: false,
-          sort: sortValue
+          sort: { value: sortValue, order: 'DESC' },
```

Comment:
Since we're introducing sorting in both orders for Active now, order here should also get `sortOrder` like History does.